--- origsrc/folks-0.4.3/backends/key-file/Makefile.am	2011-03-21 12:51:00.000000000 -0500
+++ src/folks-0.4.3/backends/key-file/Makefile.am	2011-05-25 02:21:11.544249100 -0500
@@ -41,7 +41,7 @@ libfolks_backend_key_file_la_LIBADD = \
 	$(top_builddir)/folks/libfolks.la \
 	$(NULL)
 
-libfolks_backend_key_file_la_LDFLAGS = -shared -fPIC -module -avoid-version
+libfolks_backend_key_file_la_LDFLAGS = -module -avoid-version -no-undefined
 
 GITIGNOREFILES = \
 	folks-backend-key-file.vapi \
--- origsrc/folks-0.4.3/backends/telepathy/Makefile.am	2011-03-21 12:51:00.000000000 -0500
+++ src/folks-0.4.3/backends/telepathy/Makefile.am	2011-05-25 02:21:24.020962800 -0500
@@ -51,7 +51,7 @@ libfolks_backend_telepathy_la_LIBADD = \
 	lib/libfolks-telepathy.la \
 	$(NULL)
 
-libfolks_backend_telepathy_la_LDFLAGS = -shared -fPIC -module -avoid-version
+libfolks_backend_telepathy_la_LDFLAGS = -module -avoid-version -no-undefined
 
 GITIGNOREFILES = \
 	folks-backend-telepathy.vapi \
--- origsrc/folks-0.4.3/backends/telepathy/lib/Makefile.am	2011-04-26 12:32:17.000000000 -0500
+++ src/folks-0.4.3/backends/telepathy/lib/Makefile.am	2011-05-25 02:22:37.104142900 -0500
@@ -139,6 +139,7 @@ libfolks_telepathy_la_LIBADD = \
 # that vim quickfix mode (:make) doesn't interpret the libtool --mode=link
 # command as an error message in a bizarrely named file
 libfolks_telepathy_la_LDFLAGS = \
+	-no-undefined \
 	-version-info "$(LT_CURRENT)":"$(LT_REVISION)":"$(LT_AGE)" \
 	-export-symbols-regex "^tpf_.*" \
 	$(NULL)
--- origsrc/folks-0.4.3/folks/Makefile.am	2011-04-26 12:32:17.000000000 -0500
+++ src/folks-0.4.3/folks/Makefile.am	2011-05-25 02:22:21.728263400 -0500
@@ -68,6 +68,7 @@ libfolks_la_LIBADD = \
 # that vim quickfix mode (:make) doesn't interpret the libtool --mode=link
 # command as an error message in a bizarrely named file
 libfolks_la_LDFLAGS = \
+	-no-undefined \
 	-version-info "$(LT_CURRENT)":"$(LT_REVISION)":"$(LT_AGE)"
 
 folksdir = $(includedir)/folks
