--- origsrc/folks-0.6.5/backends/eds/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/backends/eds/Makefile.am	2011-11-18 12:37:00.151009400 -0600
@@ -3,6 +3,7 @@ SUBDIRS = lib
 BACKEND_NAME = "eds"
 
 AM_CPPFLAGS = \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/backends/eds/lib \
 	-I$(top_srcdir)/backends/eds/lib/folks \
--- origsrc/folks-0.6.5/backends/eds/lib/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/backends/eds/lib/Makefile.am	2011-11-18 12:37:19.931140800 -0600
@@ -1,6 +1,7 @@
 BACKEND_NAME = "eds"
 
 AM_CPPFLAGS = \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-include $(CONFIG_HEADER) \
 	-DPACKAGE_DATADIR=\"$(pkgdatadir)\" \
--- origsrc/folks-0.6.5/backends/key-file/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/backends/key-file/Makefile.am	2011-11-18 11:12:15.054158500 -0600
@@ -1,6 +1,7 @@
 BACKEND_NAME = "key-file"
 
 AM_CPPFLAGS = \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-include $(CONFIG_HEADER) \
 	-DPACKAGE_DATADIR=\"$(pkgdatadir)\" \
--- origsrc/folks-0.6.5/backends/libsocialweb/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/backends/libsocialweb/Makefile.am	2011-11-18 12:36:38.337761800 -0600
@@ -3,6 +3,7 @@ SUBDIRS = lib
 BACKEND_NAME = "libsocialweb"
 
 AM_CPPFLAGS = \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/backends/libsocialweb/lib \
 	-I$(top_srcdir)/backends/libsocialweb/lib/folks \
--- origsrc/folks-0.6.5/backends/libsocialweb/lib/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/backends/libsocialweb/lib/Makefile.am	2011-11-18 12:36:22.281843400 -0600
@@ -1,6 +1,7 @@
 BACKEND_NAME = "libsocialweb"
 
 AM_CPPFLAGS = \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-include $(CONFIG_HEADER) \
 	-DPACKAGE_DATADIR=\"$(pkgdatadir)\" \
--- origsrc/folks-0.6.5/backends/telepathy/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/backends/telepathy/Makefile.am	2011-11-18 11:12:39.598562400 -0600
@@ -3,6 +3,7 @@ SUBDIRS = lib
 BACKEND_NAME = "telepathy"
 
 AM_CPPFLAGS = \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/backends/telepathy/lib \
 	-I$(top_srcdir)/backends/telepathy/lib/folks \
--- origsrc/folks-0.6.5/backends/telepathy/lib/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/backends/telepathy/lib/Makefile.am	2011-11-18 12:37:24.929426700 -0600
@@ -1,6 +1,7 @@
 BACKEND_NAME = "telepathy"
 
 AM_CPPFLAGS = \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-include $(CONFIG_HEADER) \
 	-DPACKAGE_DATADIR=\"$(pkgdatadir)\" \
--- origsrc/folks-0.6.5/backends/tracker/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/backends/tracker/Makefile.am	2011-11-18 12:37:11.438655000 -0600
@@ -3,6 +3,7 @@ SUBDIRS = lib
 BACKEND_NAME = "tracker"
 
 AM_CPPFLAGS = \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/backends/tracker/lib \
 	-I$(top_srcdir)/backends/tracker/lib/folks \
--- origsrc/folks-0.6.5/backends/tracker/lib/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/backends/tracker/lib/Makefile.am	2011-11-18 12:37:29.026661000 -0600
@@ -1,6 +1,7 @@
 BACKEND_NAME = "tracker"
 
 AM_CPPFLAGS = \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-include $(CONFIG_HEADER) \
 	-DPACKAGE_DATADIR=\"$(pkgdatadir)\" \
--- origsrc/folks-0.6.5/tests/eds/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/tests/eds/Makefile.am	2011-11-18 12:48:52.479752300 -0600
@@ -3,6 +3,7 @@ AM_CPPFLAGS = \
 	$(GEE_CFLAGS) \
 	$(EBOOK_CFLAGS) \
 	$(EDATASERVER_CFLAGS) \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/backends/eds/lib \
 	-I$(top_srcdir)/backends/eds/lib/folks \
--- origsrc/folks-0.6.5/tests/folks/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/tests/folks/Makefile.am	2011-11-18 12:48:55.262911500 -0600
@@ -2,6 +2,7 @@ AM_CPPFLAGS = \
 	$(GLIB_CFLAGS) \
 	$(GEE_CFLAGS) \
 	$(TP_GLIB_CFLAGS) \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/tests/lib \
 	-I$(top_srcdir)/tests/lib/key-file \
--- origsrc/folks-0.6.5/tests/key-file/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/tests/key-file/Makefile.am	2011-11-18 12:48:58.031069800 -0600
@@ -1,6 +1,7 @@
 AM_CPPFLAGS = \
 	$(GLIB_CFLAGS) \
 	$(GEE_CFLAGS) \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/tests/lib \
 	-I$(top_srcdir)/tests/lib/key-file \
--- origsrc/folks-0.6.5/tests/lib/eds/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/tests/lib/eds/Makefile.am	2011-11-18 12:45:46.009086800 -0600
@@ -28,6 +28,7 @@ libeds_test_la_CFLAGS = \
 	$(GEE_CFLAGS) \
 	$(EBOOK_CFLAGS) \
 	$(EDATASERVER_CFLAGS) \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/backends/eds/lib \
 	-I$(top_srcdir)/backends/eds/lib/folks \
--- origsrc/folks-0.6.5/tests/lib/tracker/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/tests/lib/tracker/Makefile.am	2011-11-18 12:46:02.941055200 -0600
@@ -24,6 +24,7 @@ libtracker_test_la_CFLAGS = \
 	$(GLIB_CFLAGS) \
 	$(GEE_CFLAGS) \
 	$(TRACKER_SPARQL_CFLAGS) \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/backends/tracker/lib \
 	-I$(top_srcdir)/backends/tracker/lib/folks \
--- origsrc/folks-0.6.5/tests/libsocialweb/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/tests/libsocialweb/Makefile.am	2011-11-18 12:49:02.207308700 -0600
@@ -3,6 +3,7 @@ AM_CPPFLAGS = \
 	$(GIO_CFLAGS) \
 	$(GEE_CFLAGS) \
 	$(SW_CLIENT_CFLAGS) \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/backends/libsocialweb/lib \
 	-I$(top_srcdir)/backends/libsocialweb/lib/folks \
--- origsrc/folks-0.6.5/tests/telepathy/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/tests/telepathy/Makefile.am	2011-11-18 12:49:04.744453800 -0600
@@ -2,6 +2,7 @@ AM_CPPFLAGS = \
 	$(GLIB_CFLAGS) \
 	$(GEE_CFLAGS) \
 	$(TP_GLIB_CFLAGS) \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/backends/telepathy \
 	-I$(top_srcdir)/backends/telepathy/lib \
--- origsrc/folks-0.6.5/tests/tracker/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/tests/tracker/Makefile.am	2011-11-18 12:49:08.736682100 -0600
@@ -2,6 +2,7 @@ AM_CPPFLAGS = \
 	$(GLIB_CFLAGS) \
 	$(GEE_CFLAGS) \
 	$(TRACKER_SPARQL_CFLAGS) \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	-I$(top_srcdir)/backends/tracker/lib \
 	-I$(top_srcdir)/backends/tracker/lib/folks \
--- origsrc/folks-0.6.5/tools/Makefile.am	2011-09-27 15:43:44.000000000 -0500
+++ src/folks-0.6.5/tools/Makefile.am	2011-11-18 13:03:19.081319100 -0600
@@ -32,6 +32,7 @@ folks_import_CFLAGS = \
 	$(GLIB_CFLAGS) \
 	$(GEE_CFLAGS) \
 	$(LIBXML_CFLAGS) \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/folks \
 	$(NULL)
 folks_import_LDADD = \
