--- origsrc/folks-0.6.5/backends/eds/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/backends/eds/Makefile.am	2011-11-18 12:37:00.151009400 -0600
@@ -59,7 +59,7 @@ eds_la_LIBADD = \
 	$(LIBXML_LIBS) \
 	$(NULL)
 
-eds_la_LDFLAGS = -shared -fPIC -module -avoid-version
+eds_la_LDFLAGS = -shared -fPIC -module -avoid-version -no-undefined
 
 GITIGNOREFILES = \
 	folks-backend-eds.vapi \
--- origsrc/folks-0.6.5/backends/eds/lib/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/backends/eds/lib/Makefile.am	2011-11-18 12:37:19.931140800 -0600
@@ -81,6 +81,7 @@ libfolks_eds_la_LIBADD = \
 # that vim quickfix mode (:make) doesn't interpret the libtool --mode=link
 # command as an error message in a bizarrely named file
 libfolks_eds_la_LDFLAGS = \
+	-no-undefined \
 	-version-info "$(LT_CURRENT)":"$(LT_REVISION)":"$(LT_AGE)" \
 	-export-symbols-regex "^(EDSF|edsf)_.*|" \
 	$(NULL)
--- origsrc/folks-0.6.5/backends/key-file/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/backends/key-file/Makefile.am	2011-11-18 11:12:15.054158500 -0600
@@ -41,7 +41,7 @@ key_file_la_LIBADD = \
 	$(top_builddir)/folks/libfolks.la \
 	$(NULL)
 
-key_file_la_LDFLAGS = -shared -fPIC -module -avoid-version
+key_file_la_LDFLAGS = -shared -fPIC -module -avoid-version -no-undefined
 
 GITIGNOREFILES = \
 	folks-backend-key-file.vapi \
--- origsrc/folks-0.6.5/backends/libsocialweb/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/backends/libsocialweb/Makefile.am	2011-11-18 12:36:38.337761800 -0600
@@ -53,7 +53,7 @@ libsocialweb_la_LIBADD = \
 	lib/libfolks-libsocialweb.la \
 	$(NULL)
 
-libsocialweb_la_LDFLAGS = -shared -fPIC -module -avoid-version
+libsocialweb_la_LDFLAGS = -shared -fPIC -module -avoid-version -no-undefined
 
 GITIGNOREFILES = \
 	folks-backend-libsocialweb.vapi \
--- origsrc/folks-0.6.5/backends/libsocialweb/lib/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/backends/libsocialweb/lib/Makefile.am	2011-11-18 12:36:22.281843400 -0600
@@ -74,6 +74,7 @@ libfolks_libsocialweb_la_LIBADD = \
 
 
 libfolks_libsocialweb_la_LDFLAGS = \
+	-no-undefined \
 	-version-info "$(LT_CURRENT)":"$(LT_REVISION)":"$(LT_AGE)" \
         -export-symbols-regex "^swf_.*" \
 	$(NULL)
--- origsrc/folks-0.6.5/backends/telepathy/Makefile.am	2011-11-02 11:00:02.000000000 -0500
+++ src/folks-0.6.5/backends/telepathy/Makefile.am	2011-11-18 11:12:39.598562400 -0600
@@ -53,7 +53,7 @@ telepathy_la_LIBADD = \
 	lib/libfolks-telepathy.la \
 	$(NULL)
 
-telepathy_la_LDFLAGS = -shared -fPIC -module -avoid-version
+telepathy_la_LDFLAGS = -shared -fPIC -module -avoid-version -no-undefined
 
 GITIGNOREFILES = \
 	folks-backend-telepathy.vapi \
--- origsrc/folks-0.6.5/backends/telepathy/lib/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/backends/telepathy/lib/Makefile.am	2011-11-18 12:37:24.929426700 -0600
@@ -144,6 +144,7 @@ libfolks_telepathy_la_LIBADD = \
 libfolks_telepathy_la_LDFLAGS = \
 	-version-info "$(LT_CURRENT)":"$(LT_REVISION)":"$(LT_AGE)" \
 	-export-symbols-regex "^tpf_.*" \
+	-no-undefined \
 	$(NULL)
 
 folks_telepathy_includedir = $(includedir)/folks
--- origsrc/folks-0.6.5/backends/tracker/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/backends/tracker/Makefile.am	2011-11-18 12:37:11.438655000 -0600
@@ -54,7 +54,7 @@ tracker_la_LIBADD = \
 	lib/libfolks-tracker.la \
 	$(NULL)
 
-tracker_la_LDFLAGS = -shared -fPIC -module -avoid-version
+tracker_la_LDFLAGS = -shared -fPIC -module -avoid-version -no-undefined
 
 GITIGNOREFILES = \
 	folks-backend-tracker.vapi \
--- origsrc/folks-0.6.5/backends/tracker/lib/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/backends/tracker/lib/Makefile.am	2011-11-18 12:37:29.026661000 -0600
@@ -76,6 +76,7 @@ libfolks_tracker_la_LIBADD = \
 # that vim quickfix mode (:make) doesn't interpret the libtool --mode=link
 # command as an error message in a bizarrely named file
 libfolks_tracker_la_LDFLAGS = \
+	-no-undefined \
 	-version-info "$(LT_CURRENT)":"$(LT_REVISION)":"$(LT_AGE)" \
 	-export-symbols-regex "^(TRF|trf)_.*|" \
 	$(NULL)
--- origsrc/folks-0.6.5/folks/Makefile.am	2011-11-10 16:36:54.000000000 -0600
+++ src/folks-0.6.5/folks/Makefile.am	2011-11-18 11:12:57.004557900 -0600
@@ -120,6 +120,7 @@ libfolks_la_LIBADD = \
 # that vim quickfix mode (:make) doesn't interpret the libtool --mode=link
 # command as an error message in a bizarrely named file
 libfolks_la_LDFLAGS = \
+	-no-undefined \
 	-version-info "$(LT_CURRENT)":"$(LT_REVISION)":"$(LT_AGE)"
 
 folksdir = $(includedir)/folks
